themes/classic/templates/checkout/_partials/order-confirmation-table.tpl

```diff
@@ -13,7 +13,9 @@
           </span>
         </td>
         <td>
-          {$product.name}
+          {if isset($is_final_summary)}<a href="{$product.url}" target="_blank">{/if}
```

Comment:
Can you remove the `isset()` too and define the `is_final_summary` variable in all the includes, please?
Also, if you set it to false, the `isset()` will return false and I guess it's not what you want.